README.md

```diff
@@ -42,25 +42,28 @@ There are some command line arguments that you can set when running the gateway:
 
 ## Docker
 
-A [Dockerfile](./Dockerfile) is provided, together with a [docker-compose.yml](./docker-compose.yml) file to run the gateway. Make sure you have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
+A [Dockerfile](./Dockerfile) is provided, together with the files [docker-compose.yml](./docker-compose.yml), [docker-compose-local.yml](./docker-compose-local.yml) and [docker-compose-prod.yml](./docker-compose-prod.yml) to run the gateway according to the needs. Make sure you have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
 
 A Docker image is also available at [omniadevs/ic-websocket-gateway](https://hub.docker.com/r/omniadevs/ic-websocket-gateway). This is the image used in the [docker-compose.yml](./docker-compose.yml) file.
 
 To run the gateway with Docker Compose, follow these steps:
```

Comment:
I think we should have two separate sub-sections that explain how to run it locally and in production, because:
- point 1. applies only to production
- point 2. and 3. apply only to local
- point 4. and 5. apply only to production
- point 6. should be explicitly explained for both local and production
- point 7. applies to both
- the final explanation is something that you'd likely do only in the local env